dirs/dirs.go

```diff
@@ -298,6 +299,7 @@ func SetRootDir(rootdir string) {
 	SnapRollbackDir = filepath.Join(rootdir, snappyDir, "rollback")
 
 	SnapBinariesDir = filepath.Join(SnapMountDir, "bin")
+	SnapManpagesDir = filepath.Join(SnapMountDir, "man")
```

Comment:
A random, thought, should we block installation of snaps named `bin` and `man` like we do for `system` ?

Reply:
CC @jdstrand for review tools. I think we should do a separate pass of checks in snapd though.